ExampleMod/Content/Items/Tools/ExampleHook.cs

```diff
@@ -100,7 +101,7 @@ internal class ExampleHookProjectile : ModProjectile
 			speed = 10;
 		}
 
-		private Texture2D chainTexture = GetTexture("ExampleMod/Content/Items/Tools/ExampleHookChain").Value;
+		private Asset<Texture2D> chainTexture = GetTexture("ExampleMod/Content/Items/Tools/ExampleHookChain");
```

Comment:
someone teach me how to use this asset system properly someday

Reply:
So much better

Reply:
Should've just commited this directly outside the PR, but whatever.